src/Illuminate/Foundation/Console/ConfigDumpCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Illuminate\Foundation\Console;
+
+use Illuminate\Config\Repository;
+use Illuminate\Console\Command;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Collection;
+
+class ConfigDumpCommand extends Command
+{
+    /**
+     * @var string
+     */
+    protected $signature = 'config:dump {id?}';
```

Comment:
maybe we should use `key` is here?

Reply:
Maybe. But i focused on it:
https://github.com/php-fig/container/blob/master/src/ContainerInterface.php#L22